sklearn/mixture/dpgmm.py

```diff
@@ -501,10 +501,10 @@ def fit(self, X):
             List of n_features-dimensional data points.  Each row
             corresponds to a single data point.
         """
-        self.random_state = check_random_state(self.random_state)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
BTW, wouldn't it make sense to have a common test to check that `self.random_state` is not mutated by a call to `fit` when the constructor parameter is passed with an integer seed?